code/polymorphism/solution/trypoly.sol.cpp

```diff
@@ -6,11 +6,11 @@ int main() {
     Pentagon penta{1.0};
     std::cout << "Penta : perimeter = " << penta.computePerimeter() << "\n\n";
 
-    // create an Hexagon, call its perimeter method
+    // create a Hexagon, call its perimeter method
     Hexagon hexa{1.0};
     std::cout << "Hexa : perimeter = " << hexa.computePerimeter() << "\n\n";
 
-    // create an Hexagon, call its parent's perimeter method
+    // create a Hexagon, call the perimeter method through a reference to Polygon
```

Comment:
This should steer students in the right direction and not write `Hexagon h{2.0f}; h.Polygon::computePerimeter();`.